src/olympia/devhub/templates/devhub/addons/listing/items.html

```diff
@@ -13,6 +13,10 @@
         <p class="incomplete">
           {{ _("This add-on doesn't have any versions.")}}
         </p>
+      {% elif addon.status == amo.STATUS_NULL and addon.current_beta_version %}
+          <p class="incomplete">
+            {{ _("This add-on doesn't have any listed versions, so its public pages (including beta versions) are hidden.") }}
```

Comment:
hmm... I'm not sure about this copy.  "... doesn't have any listed versions" is ambiguous at least (if not incorrect) if you think about how we talk about listed and unlisted versions (it **does** have listed versions).  "... doesn't have any approved versions"?